Realm/Realm.Unity/Editor/Common/Analytics.cs

```diff
@@ -63,6 +64,7 @@ internal class Analytics
       ""Binding"": ""dotnet"",
       ""Language"": ""c#"",
       ""Framework"": ""xamarin"",
+      ""Sync Enabled"": ""%SYNC_ENABLED%"",
```

Comment:
I checked that back when I started working on this issue.
In the java sdk they check if it's enabled and this information is used to fill the field "Sync Version" accordingly. Since when asked, you said that we don't/can't show in an easy way the version of sync I just created that "Sync Enabled" field.

You can see this at line 66 for the field definition in json and line 102 for the string substitution
https://github.com/realm/realm-java/blob/9e0b932ba626b2094ac67469e68cec7d94827db1/realm-transformer/src/main/java/io/realm/transformer/RealmAnalytics.java

Reply:
I see - it's a bit of a bummer that they chose not to report it as a bool field, but 🤷‍♂️